### PR TITLE
Issue 131

### LINKED ISSUE #131: Update with 7.x-2.0-beta6
https://www.drupal.org/project/feeds/releases/7.x-2.0-beta6

### PR DESCRIPTION
**Pull request by herbdool**
Fixes #131

### CODE REVIEW

File: tests/feeds/multilingual_en_fr.csv [hunk 1]
```diff
@@ -1 +1,2 @@
...
\ No newline at end of file
+"guid","title_en","title_fr","body_en","body_fr","date_en","date_fr","datestamp_en","datestamp_fr","datetime_en","datetime_fr","image_en","image_fr","image_alt_en","image_alt_fr","image_title_en","image_title_fr","link_en","link_fr","list_boolean_en","list_boolean_fr","number_decimal_en","number_decimal_fr","number_float_en","number_float_fr","number_integer_en","number_integer_fr","term_en","term_fr","text_en","text_fr"
+1,"Testing Multilingual Feeds 1","Teste Feeds Multilingue 1","This is the body","Ceci est la corps","21-10-2015","05-11-1955",1445470140,-446731187,"21-10-2015 23:29","1955-11-05 12:00:13","public://images/foosball.jpeg","public://images/la fayette.jpeg","Foosball","La Fayette","Foosball played by two guys","la Fayette dans les bois","http://google.ca","http://google.fr",0,1,4.2,1.2,3.1416,5.6295,1000,2000,"News","Nouvelles","Carrots","Carottes"
```
Comment (yorkshire-pudding) [inline tests/feeds/multilingual_en_fr.csv:2]: Add new line at end
```suggestion
1,"Testing Multilingual Feeds 1","Teste Feeds Multilingue 1","This is the body","Ceci est la corps","21-10-2015","05-11-1955",1445470140,-446731187,"21-10-2015 23:29","1955-11-05 12:00:13","public://images/foosball.jpeg","public://images/la fayette.jpeg","Foosball","La Fayette","Foosball played by two guys","la Fayette dans les bois","http://google.ca","http://google.fr",0,1,4.2,1.2,3.1416,5.6295,1000,2000,"News","Nouvelles","Carrots","Carottes"

```

File: tests/feeds/multilingual_en_fr_empty.csv [hunk 2]
```diff
@@ -1 +1,2 @@
...
\ No newline at end of file
+"guid","title_en","title_fr","body_en","body_fr","date_en","date_fr","datestamp_en","datestamp_fr","datetime_en","datetime_fr","image_en","image_fr","image_alt_en","image_alt_fr","image_title_en","image_title_fr","link_en","link_fr","list_boolean_en","list_boolean_fr","number_decimal_en","number_decimal_fr","number_float_en","number_float_fr","number_integer_en","number_integer_fr","term_en","term_fr","text_en","text_fr"
+1,,"Teste Feeds Multilingue 1",,"Ceci est la corps",,"05-11-1955",,-446731187,,"1955-11-05 12:00:13",,"public://images/la fayette.jpeg",,"La Fayette",,"la Fayette dans les bois",,"http://google.fr",,1,,1.2,,5.6295,,2000,,"Nouvelles",,"Carottes"
```
Comment (yorkshire-pudding) [inline tests/feeds/multilingual_en_fr_empty.csv:2]: Add new line at end
```suggestion
1,,"Teste Feeds Multilingue 1",,"Ceci est la corps",,"05-11-1955",,-446731187,,"1955-11-05 12:00:13",,"public://images/la fayette.jpeg",,"La Fayette",,"la Fayette dans les bois",,"http://google.fr",,1,,1.2,,5.6295,,2000,,"Nouvelles",,"Carottes"

```

**Review by yorkshire-pudding — CHANGES_REQUESTED** [review]
@herbdool - I'm guessing you requested a review from me as I've previously raised PHP 8.1 issues.  As far as I understand the purpose of the changes, this looks mostly ok though I've suggested new lines at the end of the CSV files.

I've used feed but don't think I've ever had need to use Simple Pie so I've really just reviewed on coding standards and whether it makes sense.

File: tests/feeds/multilingual_en_fr.csv (hunk 1 above)
Comment (herbdool) [reply tests/feeds/multilingual_en_fr.csv:2]: I think the importer would treat the new line as a row for the importer. At least that's the issue I've had with another app. So I'm inclined to leave it out.

**Comment by herbdool** [comment]
@yorkshire-pudding thanks for looking it over.

**Pull request merged**